_posts/2000-01-01-intro.md

```diff
@@ -10,11 +10,11 @@ style: center
 
 <div markdown="1">
 
-[![Build Status](https://travis-ci.org/mockito/mockito.svg?branch=release/2.x)](https://travis-ci.org/mockito/mockito)
+[![Build Status](https://travis-ci.org/mockito/mockito.svg?branch=release/3.x)](https://travis-ci.org/mockito/mockito)
 [![Coverage Status](https://img.shields.io/codecov/c/github/mockito/mockito.svg)](https://codecov.io/github/mockito/mockito)
 [![MIT License](https://img.shields.io/badge/license-MIT-green.svg)](https://github.com/mockito/mockito/blob/master/LICENSE)
 
-[![Bintray](https://api.bintray.com/packages/mockito/maven/mockito-development/images/download.svg)](https://bintray.com/mockito/maven)
```

Comment:
This is technically correct, as it is our development version. However, we should probably instruct Shipkit to publish to both Bintray `mockito` and `mockito-development`. CC @mockitoguy 

Can we keep `mockito-development` here for now?

Reply:
I've removed the badge update, it's a whole separate topic that should be addressed separately.